.github/workflows/release.yaml

```
@@ -0,0 +1,39 @@
+---
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to be released'
+        required: true
+
+env:
+  REGISTRY_NAME: registry.scality.com
+  PROJECT_NAME: ${{ github.event.repository.name }}
+
+jobs:
+  build-federation-image:
+    uses: scality/workflows/.github/workflows/docker-build.yaml@v1
+    secrets: inherit
+    with:
+      push: true
+      registry: ${{ env.REGISTRY_NAME }}
+      namespace: ${{env.PROJECT_NAME}}
+      name: ${{env.PROJECT_NAME}}
```

Argh, can't believe I'm being this person 😂 
```suggestion
      namespace: ${{env.PROJECT_NAMEv}}
      name: ${{env.PROJECT_NAME }}
```

---

Oh missing spaces yea
```suggestion
      namespace: ${{ env.PROJECT_NAME }}
      name: ${{ env.PROJECT_NAME }}
```
You mean this?

---

Ah, I missed the ones at the beginning 😂 

Yes!

---

and I'm noticing I included a `v` ?? 🤦 My bad